### PR TITLE
Removed stdin from the input handler.

### PR DESCRIPTION
**Pull request by efrisch**
Fixes pyrsia/pyrsia#349

### CODE REVIEW

**Comment by MitaliBo** [comment]
Lgtm, #356 will follow this to have more structured body for the api request and response and integrated with CLI

**Review by MitaliBo — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -131,6 +131,11 @@ the Pyrsia node is running.
     In a real life deployment these nodes will be spread over the network and will all run on their own 7888 port.
     `Word of caution: Running the peers for a few hours does generate network traffic and hence can drain your computer power. Ensure you are plugged into power if you are running multiple peers for a long time`
 
+    To test the pyrsia_node status you can use `curl`  and
+    ```
+    curl --location --request GET 'http://localhost:7888/status'
```
Comment (erwin1) [inline readme.md:136]: @efrisch I'm not opposed to a status endpoint, but isn't the CLI better suited for this?

Comment (prince-chrismc) [reply readme.md:136]: Great question 🤔 How is this different from 

image

or is this just a more direct manual way of testing?

Comment (efrisch) [reply readme.md:136]: Just documenting a more direct manual way of testing. Yes, they are equivalent. We talked about including postman files too.

**Review by erwin1 — APPROVED** [review]

**Review by MitaliBo — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Excellent team work 🎉

**Pull request merged**